applications/nrf5340_audio/broadcast_source/main.c

```diff
@@ -312,11 +317,67 @@ void streamctrl_send(void const *const data, size_t size, uint8_t num_ch)
 	}
 }
 
+#if CONFIG_CUSTOM_BROADCASTER
+/* Example of how to create a custom broadcaster */
+/**
+ * Remember to increase:
+ * CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT
+ * CONFIG_BT_CTLR_ADV_ISO_STREAM_COUNT (set in hci_ipc.conf)
+ * CONFIG_BT_ISO_TX_BUF_COUNT
+ * CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT
+ * CONFIG_BT_ISO_MAX_CHAN
+ */
+#error Feature is incomplete and should only be used as a guideline for now
+static struct bt_bap_lc3_preset lc3_preset_48 = BT_BAP_LC3_BROADCAST_PRESET_48_4_1(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static void broadcast_create(struct broadcast_source_big *broadcast_param)
+{
+	static enum bt_audio_location location[2] = {BT_AUDIO_LOCATION_FRONT_LEFT,
+						     BT_AUDIO_LOCATION_FRONT_RIGHT};
+	static struct subgroup_config subgroups[2];
+
+	subgroups[0].group_lc3_preset = lc3_preset_48;
+	subgroups[0].num_bises = 2;
+	subgroups[0].context = BT_AUDIO_CONTEXT_TYPE_MEDIA;
+	subgroups[0].location = location;
+
+	subgroups[1].group_lc3_preset = lc3_preset_48;
+	subgroups[1].num_bises = 2;
+	subgroups[1].context = BT_AUDIO_CONTEXT_TYPE_MEDIA;
+	subgroups[1].location = location;
+
+	broadcast_param->packing = BT_ISO_PACKING_INTERLEAVED;
+
+	broadcast_param->encryption = false;
+
+	bt_audio_codec_cfg_meta_set_bcast_audio_immediate_rend_flag(
+		&subgroups[0].group_lc3_preset.codec_cfg);
+	bt_audio_codec_cfg_meta_set_bcast_audio_immediate_rend_flag(
+		&subgroups[1].group_lc3_preset.codec_cfg);
+
+	uint8_t spanish_src[3] = "spa";
+	uint8_t english_src[3] = "eng";
+
```

Comment:
Same as above, won't use defines for this code since it is only temporary here. Partly because of the new main file that is coming and partly to show what is possible to do with this API. There is no define for `eng` in Zephyr, and I guess part of the reason is that ISO 639-3 contains roughly 520 entries just for the letter 'A'...